From 42573d84b28b82eed76cbac7848e7430fecd0cc9 Mon Sep 17 00:00:00 2001
From: dkl <dkl@rock-chips.com>
Date: Wed, 25 Jun 2014 20:21:16 +0800
Subject: [PATCH] rk3288: set "rockchip,clocks-uboot-has-init" "aclk_vio0" as
 default.

This commit corresponds to commit debf1d2237185a26503d737d54db730f62cf5ea5.
---
 arch/arm/boot/dts/rk3288.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3288.dtsi b/arch/arm/boot/dts/rk3288.dtsi
index d597faabddd0..14f2a108bcd9 100755
--- a/arch/arm/boot/dts/rk3288.dtsi
+++ b/arch/arm/boot/dts/rk3288.dtsi
@@ -496,7 +496,7 @@
 			<&clk_isp_jpe 400000000>, <&clk_tsp 80000000>,
 			<&clk_tspout 80000000>, <&clk_mac 125000000>;
 		rockchip,clocks-uboot-has-init =
-			<&aclk_vio1>;
+			<&aclk_vio0>;
 	};
 
 	clocks-enable {
-- 
2.35.3

